modules/microshift-disabling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+// * microshift_configuring/microshift-disable-lvms-csi-provider-csi-snapshot.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disabling-lvms-csi-snapshot_{context}"]
+= Disabling deployments that run CSI snapshot implementations
+
+Before you disable the CSI snapshot implementation pods, you must manually stop any user workloads that have attached volumes. Use the following procedure to disable installation of the CSI implementation pods.
+
+[NOTE]
+====
+{microshift-short} will not delete CSI snapshot implementation pods. You must configure {microshift-short} to disable installation of the CSI snapshot implementation pods during the startup process.
+====
+
+.Procedure
+
+. Disable installation of the CSI snapshot controller by entering the `optionalCsiComponents` value under the `storage` section of the {microshift-short} configuration file:
++
+[source,yaml]
+----
+# ...
+  storage
+   optionalCsiComponents: "csi-snapshot-controller" <1>
```

Comment:
The value of `csi-snapshot-controller` throws an error when restarting MicroShift.

The `config.yaml/default` entry looks like this

`   # Allowed values are: unset, [], or one or more of ["snapshot-controller", "snapshot-webhook"]

I think `csi` is not needed prepended to the component. I would check with development

Reply:
@copejon Please confirm

Reply:
correct

Reply:
done. csi is removed as suggested